tests/packages/extra/test_extra_group_pkgs.py

```diff
@@ -0,0 +1,12 @@
+from lib.commands import SSHCommandFailed
+from pkgfixtures import host_with_saved_yum_state
```

Comment:
Adding the usual comment for package-scoped fixtures, since this explicit import should not be necessary if we weren't workarounding a pytest issue.

```suggestion

# Explicitly import package-scoped fixtures (see explanation in pkgfixtures.py)
from pkgfixtures import host_with_saved_yum_state
```